USING.md

```diff
@@ -47,6 +58,10 @@ With local installation:
 ```sh
 zonemaster-cli zonemaster.net
 ```
+or
+```sh
+zonemaster-cli zonemaster.net --no-ipv6
+```
```

Comment:
Again, I would suggest to only keep the default (and possibly most used) command and let users tweak it to their needs.

Reply:
Can we leave as-is for now, and tweak for next release? If so, can you approve?